### ISSUE TITLE
Updating to LCR-Froyo-1.1 (patch): Navigator's Volume returned back to LOW!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dkuspawono/acer-liquid-community-rom-bugtracker#208]
```
!!!PLEASE WIPE BEFORE POSTING A BUG!!!


What steps will reproduce the problem?
1. Update from LCR-F 1.0 to 1.1
2.
3.

What is the expected output? What do you see instead?
The same volume for Navigator, instead it decreased a lot!
What is the bin file you are using ? (0.010, 0.014, 0.016, 1.100...)
Acer_LiquidE_4.504b.00_EMEA-GEN1
What version are you using ?

What patch or addons pack have you installed ?

If you use LCR 1.7 attach le file LCR_UI.txt stored on your sdcard or
answer :
|THIS_CONF_MUST_BE_APPLIED| ?? |
|RECOVERY| ?? |
|FULL_UI_ACER| ?? |
|FULL_UI_ANDROID| ?? |
|STATUS_BAR_AT_THE_BOTTOM| ?? |
|NOTIFICATION_TYPE_STREAM| ?? |
|STREAM_NOTIFICATION_ON_TOP| ?? |
|DIALER_TYPE_STREAM| ?? |
|LOCK_TYPE_STREAM| ?? |
|LAUNCHER_TYPE_STREAM| ?? |


Where are stored your apps (phone/sd default is phone) ?
sd
Where is stored your dalvik-cache (phone/sd default is phone) ?
phone
Where are stored your datas (phone/sd default is phone) ?
phone
Did you wipe dalvik ?
no, just update
Did you wipe data (factory reset)
no, just update

Please attach a log made with sendlog or log_debug tool (see wiki)


Please provide any additional information below.

Please malez correct this problem.. It's very hard to listen it!
```

Original issue reported on code.google.com by `Alezzan...@gmail.com` on 31 Oct 2010 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Confirmed!! Fixed in 1.1.1!! Thanks malez, you rock! :-) 
```

Original comment by `Alezzan...@gmail.com` on 7 Nov 2010 at 10:55
